elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/dsl/MappingDsl.scala

```diff
@@ -24,11 +24,13 @@ trait MappingDsl extends DslCommons {
     val option: String
   }
 
-  // String datatype - https://www.elastic.co/guide/en/elasticsearch/reference/current/string.html
-  case object StringType extends FieldType {
-    val rep = "string"
+  case object KeywordType extends FieldType {
+    val rep = "keyword"
   }
 
+  case object TextType extends FieldType {
```

Comment:
Added elasticsearch doc link to TextType ?

Reply:
Users can google the document for data type? I don't think we need to add the link explicitly to the code.